src/assets/sass/components/_chart.scss

```diff
@@ -30,6 +30,30 @@ ccl-chart {
     padding-top: 1.5rem;
   }
 
+  .chart-legend {
+    padding: 2rem;
+    padding-top: 0;
+    color: #757575;
+    font-size: 11px;
+
+    .chart-legend-line {
+      color: #1E8CEC;
+    }
+
+    .chart-legend-icons {
+       padding-bottom: 0.5rem;
+    }
+
+    .chart-legend-minmax {
+      color: darken(#EFF4F7, 10%);
+
+      &:before {
+        padding-left: 0.5em;
+        content: '\2B1B'; // unicode large filled square
```

Comment:
Fixed in https://github.com/azavea/climate-change-lab/pull/193/commits/760b431c3865a0de0c0b92015bd56209b1707cd8. Works fine in Firefox. I can't test in Chrome due to problems proxying through my laptop at the office.